packages/langium/test/grammar/grammar-validator.test.ts

```diff
@@ -200,6 +200,24 @@ describe('Langium grammar validation', () => {
         const validationResult = await validate(grammar);
         expectNoIssues(validationResult);
     });
+
```

Comment:
It would be nice to have an additional test that test a false positive when no regex flags are used 
```suggestion
    test('Composite terminal no regex flags', async () => {
        const grammar = `
        terminal Test: 'Test' /test/;
        `;
        const validationResult = await validate(grammar);
        expectNoIssues(validationResult);
    });

```